src/gp/src/GPMSA.C

```diff
@@ -1155,6 +1160,8 @@ GPMSAFactory<V, M>::setUpHyperpriors()
       V y_temp(Wy * y);
       y_temp -= Wy * B * yhat;
 
+      // At this point everything has already been normalized, so no
+      // more normalization to do?
```

Comment:
I'm a little surprised that we don't have to scale the experiment covariance information provided by the user, as we did in examples/gp/linear_verif/  Or maybe you are with Wy above...